prep-css-layout-classes/index.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+
+
+<head>
+
+  <meta charset="UTF-8">
+
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+  <title>CSS Layout Classes</title>
+
+  <link rel="stylesheet" href="reset.css">
+
+  <style>
+    /* stylelint-disable block-no-empty */
+
+    body {
+
+      font-family: sans-serif;
+
+    }
+
+    h1 {
+
+      text-align: center;
+
+    }
+
+    .container {
+
+      max-width: 992px;
+
+      margin: auto;
+
+      justify-content: center;
+
+      align-items: center;
+
+    }
+
+    .row {
+
+      width: 100%;
+
+      display: flex;
+
+
+
+    }
+
+    .column {
+
+      flex-basis: calc(100%/3);
+
+      padding-left: 8px;
+
+      padding-right: 8px;
+
+    }
+
+    img {
+
+      width: 100%;
+
+      height: 200px;
+
+    }
+  </style>
+
+</head>
+
+<body>
+<h1>Batman Triology</h1>
+
+    <div class="container">
+
+      <div class="row">
+
+        <div class="column">
+
+          <img
+            src="https://images-na.ssl-images-amazon.com/images/S/cmx-images-prod/Item/176903/DIG049960_2._SX1600_QL80_TTD_.jpg"
+            alt="Batman Begins">
+
+          <h3 style="color:grey;">Batman Begins</h3>
+
+          <p>
+
+            A young Bruce Wayne travels to the Far East where he's trained in the martial arts by Henri Ducard
+
+            a member of the mysterious League of Shadows. When Ducard reveals the League's true purpose -- the
+
+            complete destruction of Gotham City -- Wayne returns to Gotham intent on cleaning up the city without
+
+            resorting to murder. With the help of Alfred his loyal butler and Lucius Fox a tech expert at
+
+            Wayne Enterprises, Batman is born
+
+          </p>
+
+        </div>
+
+
+
+        <div>
+
+          <div class="container">
+
+            <div class="row">
+
+              <div class="column">
+
```

Comment:
Hi Bo, for this one you'll want to have only one container and one row, with all three columns inside the row.